client.cfg

```diff
@@ -80,19 +80,25 @@ gcode:
 description: Pause the actual running print
 rename_existing: PAUSE_BASE
 gcode:
+  SET_GCODE_VARIABLE MACRO=RESUME VARIABLE=last_extruder_temp VALUE="{printer[printer.toolhead.extruder].target}"
+
   PAUSE_BASE
   _TOOLHEAD_PARK_PAUSE_CANCEL {rawparams}
 
 [gcode_macro RESUME]
 description: Resume the actual running print
 rename_existing: RESUME_BASE
+variable_last_extruder_temp: 0
 gcode:
   ##### get user parameters or use default #####
   {% set macro_found = True if printer['gcode_macro _CLIENT_VARIABLE'] is defined else False %}
   {% set client = printer['gcode_macro _CLIENT_VARIABLE'] %}
   {% set velocity = printer.configfile.settings.pause_resume.recover_velocity %}
   {% set sp_move        = velocity if not macro_found else client.speed_move|default(velocity) %}
   ##### end of definitions #####
+  SET_HEATER_TEMPERATURE HEATER=extruder TARGET={last_extruder_temp}
+  TEMPERATURE_WAIT SENSOR=extruder MINIMUM={last_extruder_temp*0.98} MAXIMUM={last_extruder_temp*1.02}
```

Comment:
On my printer, I do prefer the solution with the target temperature window (because for me its only 3-5 degree overshoot) These are normally gone a few seconds after resuming the print. 
But it's OK with me to use M109 instead. 

My basic intention was to automatically cool down and resume the extruder temperature on idle timeout. This is still the guaranteed. So its fine!

Thanks a lot
freakyDude